plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/struct/JDBCTableColumn.java

```diff
@@ -181,7 +181,7 @@ public List<DBDLabelValuePair> getValueEnumeration(
         boolean formatValues,
         boolean caseInsensitiveSearch) throws DBException
     {
-        final String identifier = DBUtils.getQuotedIdentifier(this, DBPAttributeReferencePurpose.DATA_SELECTION);
+        String identifier = DBUtils.getQuotedIdentifier(this, DBPAttributeReferencePurpose.DATA_SELECTION);
```

Comment:
I'd return `final` modifier back. I don't get why it was removed. If it was intentionally, could you elaborate please?

Reply:
I modify indentifier later in code - thats why

Reply:
Yeah, I missed it. Thank you for explanations.